src/modules/Dashboard/features/Respondents/Popups/EditRespondentPopup/EditRespondentPopup.types.ts

```diff
@@ -3,6 +3,7 @@ import { ChosenAppletData } from '../../Respondents.types';
 export type EditRespondentForm = {
   secretUserId: string;
   nickname?: string;
+  tag?: string;
```

Comment:
```suggestion
  tag?: ParticipantTag;
```
(or `UserSelectableParticipantTag`, with my changes?)

Reply:
Yep — I was waiting for your changes so I could incorporate them into this PR. The branch is now based of your commit that updates the types, and I've changed them accordingly, here.